modules/images-other-jenkins-kubernetes-plugin.adoc

```diff
@@ -8,6 +8,13 @@
 
 In the following example, the `openshift-jee-sample` `BuildConfig` object causes a Jenkins Maven agent pod to be dynamically provisioned. The pod clones some Java source code, builds a WAR file, and causes a second `BuildConfig`, `openshift-jee-sample-docker` to run. The second `BuildConfig` layers the new WAR file into a container image.
 
+[IMPORTANT]
+====
+{product-title} 4.11 removed the OpenShift Jenkins Maven and NodeJS Agent images from its payload. Red Hat no longer produces these images, and they are not available from the `ocp-tools-4` repository at `registry.redhat.io`. Red Hat maintains the 4.10 and earlier versions of these images for any significant bug fixes or security CVEs, following the link:https://access.redhat.com/support/policy/updates/openshift[{product-title} lifecycle policy].
```

Comment:
"{product-title} 4.11 removed" sounds a bit anthropomorphic. Maybe the passive is better here:

 The OpenShift Jenkins Maven and NodeJS Agent images were removed form the {product-title} 4.11 payload.

or "are no longer part of the..." to avoid the passive.

Reply:
Updated. Thank you.